src/interfaces/resource.ts

```diff
@@ -0,0 +1,9 @@
+import {Model} from './model';
+
+export interface Resource {
+  name: string;
+  route: string;
+  description: string;
+  method: string;
+  model?: string;
```

Comment:
fix this, it isn't the real model, it's a reference to a model, and this represent the request body model